.github/workflows/ci.yaml

```diff
@@ -490,6 +425,7 @@ jobs:
 
   test-e2e:
```

Comment:
This test is not yet successful and complains about missing `libpango` @kylecarbs if you can suggest a resolution? Please see the ci workflow logs for the exact error.